plugins/performance-lab/includes/admin/plugin-activate-ajax.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Handles activation of Performance Features (Plugins) using AJAX.
+ */
+
+/* global perflabPluginActivateAjaxData */
+( function ( $ ) {
+	// @ts-ignore
+	const { i18n, a11y } = wp;
+	const { __, _x } = i18n;
+
+	/**
+	 * Adds a click event listener to the plugin activate buttons.
+	 *
+	 * @param {MouseEvent} event - The click event object that is triggered when the user clicks on the document.
+	 *
+	 * @return {Promise<void>} - The asynchronous function returns a promise.
+	 */
+	$( document ).on(
+		'click',
+		'.perflab-install-active-plugin',
+		async function ( event ) {
+			// Prevent the default link behavior.
+			event.preventDefault();
+
+			// Get the clicked element as a jQuery object.
+			const target = $( this );
+
+			target
+				.addClass( 'updating-message' )
+				.text( __( 'Activating…', 'performance-lab' ) );
+
+			a11y.speak( __( 'Activating…', 'performance-lab' ) );
+
+			// Retrieve the plugin slug from the data attribute.
+			const pluginSlug = $.trim( target.attr( 'data-plugin-slug' ) );
```

Comment:
I think we should avoid using jQuery unless we have to.

```suggestion
			const pluginSlug = target.attr( 'data-plugin-slug' ).trim();
```